src/components/Projects.tsx

```diff
@@ -1,7 +1,149 @@
 import React from "react";
+import { Link } from "react-router-dom";
+import styled from "styled-components";
+import stairs from "./assets/project-page/stairs.png";
+import shell from "./assets/project-page/knife.png";
+import eye from "./assets/project-page/eye.png";
+import statue from "./assets/project-page/statue.png";
+import dragon from "./assets/project-page/dragon.png";
+import knife from "./assets/project-page/knife.png";
+import mask from "./assets/project-page/mask.png";
+import spider from "./assets/project-page/spider.png";
+import magnify from "./assets/project-page/magnify.png";
+import Background from "./assets/home-page/background.jpg";
+import {
+  color,
+  space,
+  typography,
+  TypographyProps,
+  ColorProps,
+  SpaceProps,
+} from "styled-system";
+
+type HeadingProps = SpaceProps & ColorProps & TypographyProps;
+
+const Header = styled.div<HeadingProps>`
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  position: relative;
+`;
+
+const Title = styled.h1<HeadingProps>`
+  ${space}
+  ${color}
+  ${typography}
+  font-family: SangBleu OG Serif Light Regular;
+`;
+
+const IconContainer = styled.div`
+  width: 200px;
+  height: 120px;
+`;
+
+const Icon = styled.a`
+  a:hover {
+    position: relative;
+    width: 500px;
+    height: auto;
+  }
+`;
 
 const Projects = () => {
-  return <div>Projects</div>;
+  return (
+    <div
+      className="bg_image"
+      style={{
+        backgroundImage: "url(" + Background + ")",
```

Comment:
Ideally this would be done using styled system

Reply:
Sorry @Jbarget I mentioned to Matt yesterday but forgot to add to the PR itself, everything from the Projects Page has since been refactored using SS before these changes were dragged into the Homepage PR. Will obviously be using SS moving forward so all good on that front.

Re. string literals. Yes - my question also! If either of you can try this and let me know if works for you? Unless I'm missing something **really** obvious, VS Code doesn't like this at all. 

<img width="1076" alt="Screenshot 2020-07-03 at 13 42 27" src="https://user-images.githubusercontent.com/53219789/86466341-b839d100-bd33-11ea-9ef8-f6e43523e5f8.png">
<img width="1075" alt="Screenshot 2020-07-03 at 13 42 16" src="https://user-images.githubusercontent.com/53219789/86466343-b8d26780-bd33-11ea-9134-da379d7975b5.png">


I've tried multiple ways but `backgroundImage="url('./assets/home-page/background.jpg')"`was the only way I could get the code not to break. The issue here is that the background image does not display, any ideas?

Reply:
Sorry @Jbarget I mentioned to Matt yesterday but forgot to add to the PR itself, everything from the Projects Page has since been refactored using SS before these changes were dragged into the Homepage PR. Will obviously be using SS moving forward so all good on that front.

Re. string literals. Yes - my question also! If either of you can try this and let me know if works for you? Unless I'm missing something **really** obvious, VS Code doesn't like this at all. 

<img width="1076" alt="Screenshot 2020-07-03 at 13 42 27" src="https://user-images.githubusercontent.com/53219789/86466341-b839d100-bd33-11ea-9ef8-f6e43523e5f8.png">
<img width="1075" alt="Screenshot 2020-07-03 at 13 42 16" src="https://user-images.githubusercontent.com/53219789/86466343-b8d26780-bd33-11ea-9134-da379d7975b5.png">


I've tried multiple ways but `backgroundImage="url('./assets/home-page/background.jpg')"`was the only way I could get the code not to break. The issue here is that the background image does not display, any ideas?